src/intervalmap.rs

```diff
@@ -242,11 +251,7 @@ where
     #[inline]
     #[must_use]
     pub fn new() -> Self {
-        Self {
-            nodes: vec![Self::new_sentinel()],
-            root: Self::sentinel(),
-            len: 0,
-        }
+        Self::with_capacity(0)
```

Comment:
Why is the sentinel node removed?

Reply:
I thought we could replace it with `Node::new_sentinel` and `Node::new`, what do you think about it?

Reply:
Sorry, I didn't see it moved to the `with_capacity` method.